trees.c

```diff
@@ -733,52 +718,24 @@ static void compress_block(deflate_state *s, const ct_data *ltree, const ct_data
     unsigned dist;      /* distance of matched string */
     int lc;             /* match length or unmatched char (if dist == 0) */
     unsigned sx = 0;    /* running index in sym_buf */
-    int code;           /* the code to send */
-    int extra;          /* number of extra bits to send */
-
-    // Temp local variables
-    uint32_t bi_valid = s->bi_valid;
```

Comment:
@Dead2 perhaps this is where the speed decrease came from. The `bi_valid` and `bi_buf` variables are no longer set at the top for the loop.